src/constants.ts

```diff
@@ -13,3 +13,5 @@ export const DEFAULT_CONFIG: Config = {
   devices: [],
   exitOnPageError: true,
 }
+
+export const DEFAULT_TEST_PLAYWRIGHT_TIMEOUT = 15000
```

Comment:
maybe we should make this configurable. Because when e.g. a selector code not be found -> TimeoutError then Jest wont make a nice error reporting. Because Jest timeout < Playwright timeout.

Reply:
@mxschmitt sounds good. I should think how we can configure this.

Reply:
@mxschmitt you can configure it in **jest config** with **testTimeout**. I think that it's enought for a while